x-pack/legacy/plugins/ml/public/application/datavisualizer/index_based/page.tsx

```diff
@@ -131,7 +133,10 @@ export const Page: FC = () => {
   const defaults = getDefaultPageState();
 
   const [showActionsPanel] = useState(
```

Comment:
I think we can remove `useState` here since the value gets never updated and just do `const showActionsPanel = ...`

Reply:
Done in https://github.com/elastic/kibana/pull/55431/commits/901a1666c8250090e8c83f3565eaa85bf58d82f2